src/logic/calculate.js

```diff
@@ -0,0 +1,133 @@
+import operate from './operate';
+
+function isNumber(item) {
+  return !!item.match(/[0-9]+/);
+}
+
```

Comment:
- [ ] Kindly when I try to perform any operation, I got errors and there's no result only numbers get displayed on the screen. Kindly fix this issue.

![image](https://user-images.githubusercontent.com/35535130/176726576-1655ce89-3d03-4da9-8a7a-cc2eec82e4f8.png)